app/models/katello/purpose_status.rb

```diff
@@ -65,7 +65,11 @@ def to_status(options = {})
     end
 
     def relevant?(_options = {})
-      host.subscription_facet.try(:uuid)
```

Comment:
This can be a one-liner: `host.subscription_facet.try(:uuid) && !host.organization.simple_content_access?`

It says "It's relevant if the hosts's subscription facet has a uuid (it's registered) and the host's org is NOT in SCA mode"

Reply:
Fixed, thanks for explaining it, makes more sense now :)